packages/cli/src/cmds/preview/constants.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+"use strict";
+
+export const settingsFileName = "settings.json";
+
+export const FrontendHostingPluginName = "fx-resource-frontend-hosting";
```

Comment:
Can we get these constants from core?

Reply:
Nice idea. But neither api nor core provides the method to get these information.